decidim-participatory_processes/app/cells/decidim/participatory_processes/content_blocks/highlighted_processes_cell.rb

```diff
@@ -3,46 +3,32 @@
 module Decidim
   module ParticipatoryProcesses
     module ContentBlocks
-      class HighlightedProcessesCell < Decidim::ViewModel
-        include Decidim::ApplicationHelper
-        include Decidim::SanitizeHelper
-        include Decidim::CardHelper
-        include Cell::ViewModel::Partial
-        include ParticipatoryProcessHelper
-        include Decidim::ParticipatoryProcesses::Engine.routes.url_helpers
+      class HighlightedProcessesCell < Decidim::ContentBlocks::HighlightedParticipatorySpacesCell
+        BLOCK_ID = "highlighted-processes"
 
         delegate :current_user, to: :controller
 
-        def show
-          if single_item?
-            render "single_process"
-          elsif highlighted_items.any?
-            render
-          end
-        end
-
-        def single_item?
-          highlighted_items.length == 1
-        end
-
-        def max_results
-          model.settings.max_results
-        end
-
-        def highlighted_items
-          @highlighted_items ||= promoted_groups + highlighted_processes
+        def highlighted_spaces
+          @highlighted_spaces ||= promoted_groups + highlighted_processes
         end
+        alias limited_highlighted_spaces highlighted_spaces
```

Comment:
Add a new line here between the end and the alias. And I would move the alias above the method definitions